tools/copy_maintainers.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, name, members):
 
     @property
     def user_id(self):
+        return False
```

Comment:
Why this?

Reply:
user_id is the psc representative. We don't have one for the "virtual teams" such as OCA Member, for which this class is used. Could have hard coded Joël in there, but it did not seem useful

Reply:
Thanks for the info. A comment here would help for future references.